### ISSUE TITLE
Text rotation

### ISSUE THREAD

**Issue by ArtDepartmentMJ** [stil/gd-text#24]
Is it possible to set the text rotation?

**Comment by stil** [comment]
No, it would need some code changes. By default no rotation is set:
https://github.com/stil/gd-text/blob/master/src/Box.php#L402

**Comment by ArtDepartmentMJ** [comment]
For my purposes it was pretty easy to just create a new transparent image for the text to sit on and then just rotate that on to the canvas.

**Comment by stil** [comment]
Was text quality affected after rotation of transparent layer? I might implement it in the library, because it's probably easiest way to do this. However I'm not sure how antialiasing interferes with rotations of already rasterized text.

**Comment by daidi** [comment]
look forward to it

**Comment by daidi** [comment]
it seems we can use $angle directly .
http://php.net/manual/zh/function.imagefttext.php

**Comment by AdamBissonnette** [comment]
Any updates on this? @daidi

**Comment by daidi** [comment]
@MManifesto see this #41
